recipes/ozzy-pic/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "ozzy-pic" %}
+{% set version = "1.0.5" %}
+{% set python_min = python_min|default("3.10") %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/ozzy_pic-{{ version }}.tar.gz
+  sha256: f8a36d7c54ef98914edb6fd5d9a03b19a683c200d8a1abcd24ebc6099aaa84b5
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - poetry-core
+    - pip
+  run:
+    - python >={{ python_min }}
+    - xarray >=2024.9.0,<2025.0.0
+    - tqdm >=4.66.5,<5.0.0
+    - numba >=0.60.0,<0.61.0
+    - h5py >=3.12.1,<4.0.0
+    - h5netcdf >=1.4.0,<2.0.0
+    - flox >=0.9.12,<0.10.0
+    - dask-core >=2024.10.0,<2025.0.0
+    - dask-expr >=1.1.16,<2.0.0
+    - matplotlib-base >=3.9.2,<4.0.0
+    - cmcrameri
+    - seaborn >=0.13.2,<0.14.0
+    - ipython >=8.28.0,<9.0.0
+    - ipykernel >=6.29.5,<7.0.0
+    - hvplot >=0.11.1,<0.12.0
+    - jupyter_bokeh >=4.0.5,<5.0.0
+  run_constrained:
+    - numpy-groupies >=0.11.2,<0.12.0
+    - numbagg >=0.8.2,<0.9.0
+
+test:
+  imports:
+    - ozzy
+  commands:
+    - pip check
+  requires:
+    - python {{ python_min }}
```

Comment:
Thanks for the suggestions @ocefpaf! I did have this line before, but when I include it I get errors in the docker build tests in all three operating systems. It seems like ipython is responsible. Any ideas about how to deal with this?

I'm not sure whether you can access the error logs of the Azure automatic tests (sorry, bit of a noob here), so please let me know if I should copy the error messages to the conversation.